src/Dashboard/SponsorDashboard.tsx

```diff
@@ -0,0 +1,102 @@
+import { Box, Flex } from '@rebass/grid';
+import { AxiosResponse } from 'axios';
+import * as React from 'react';
+import { toast } from 'react-toastify';
+
+import {
+  ACCOUNT_NOT_CONFIRMED_MSG,
+  EMAIL_SENT,
+  FrontendRoute as routes,
+  RESEND_CONF_EMAIL,
+} from '../config';
+
+import { APIResponse, Auth } from '../api';
+import ValidationErrorGenerator from '../shared/Form/validationErrorGenerator';
+import WithToasterContainer from '../shared/HOC/withToaster';
+import { isConfirmed } from '../util/UserInfoHelperFunctions';
+import DashboardView, { IDashboardCard } from './View';
+
+import AccountIcon from '../assets/images/dashboard-account.svg';
+import ApplicationIcon from '../assets/images/dashboard-application.svg';
+
+export interface IDashboardState {
+  confirmed: boolean;
+}
+
+class SponsorDashboard extends React.Component<{}, IDashboardState> {
+  constructor(props: {}) {
+    super(props);
+    this.state = {
+      confirmed: true,
+    };
+  }
+
+  public async componentDidMount() {
+    try {
+      const confirmed = await isConfirmed();
+      this.setState({ confirmed });
+    } catch (e) {
+      this.setState({ confirmed: false });
+    }
+  }
+
+  public render() {
+    return (
+      <DashboardView cards={this.generateCards()} title={'Sponsor Dashboard'} />
+    );
+  }
+
+  private generateCards() {
+    const cards: IDashboardCard[] = [
+      {
+        title: 'Search',
+        route: routes.SPONSOR_SEARCH_PAGE,
+        imageSrc: ApplicationIcon,
+        validation: this.confirmAccountToastError,
+      },
+      {
+        title: 'Account',
+        route: routes.EDIT_ACCOUNT_PAGE,
+        imageSrc: AccountIcon,
+      },
+    ];
+
+    return cards;
+  }
+
+  private confirmAccountToastError = (): boolean => {
```

Comment:
Are we still doing "Confirm Account" emails for sponsors as well?

Reply:
Oh rip, we shouldn't be.